src/Cli/dotnet/Commands/Pack/PackCommandParser.cs

```diff
@@ -60,6 +64,20 @@ internal static class PackCommandParser
 
     public static readonly Option<string[]> TargetOption = CommonOptions.RequiredMSBuildTargetOption("Pack", [("_IsPacking", "true")]);
 
+    public static readonly Option<string> VersionOption = new ForwardedOption<string>("--version")
+    {
+        Description = CliCommandStrings.PackCmdVersionDescription,
+        HelpName = CliCommandStrings.PackCmdVersion,
+        Hidden = false
+    }.ForwardAsSingle(o => $"-Version={o}");
+
+    public static readonly Option<string[]> PropertiesOption = new ForwardedOption<string[]>("--properties")
+    {
+        Description = CliCommandStrings.PackCmdPropertiesDescription,
+        HelpName = CliCommandStrings.PackCmdProperties,
+        Hidden = false
+    };
```

Comment:
Strongly suggest that you reuse the existing `CommonOptions.PropertiesOption` instead here. You'll get a dictionary of key/value pairs that you can manipulate, and you generally don't want to expose `--property` and `--properties` on the same logical command - it would be confusing to the user.